src/ansys/aedt/core/filtersolutions.py

```diff
@@ -59,6 +71,11 @@ def __init__(self, version=None):
         self.transmission_zeros_bandwidth = TransmissionZeros(TableFormat.BANDWIDTH)
         self.export_to_aedt = ExportToAedt()
 
+    def close(self):
```

Comment:
Can the old design be deactivated such that continuing to use it after `close()` will result in an exception? The warning might be overlooked. Blocking use of the closed design will be more foolproof.